ansible/playbook/tasks/roles/dashboards_upgrade/tasks/main.yml

```
@@ -55,33 +55,25 @@
     path: /srv/grafana/grafana.db
   register: sqlite_grafana
 
-- name: Remove old dashboards from SQLite
-  block:
-  - name: Restart grafana before delete data
-    supervisorctl:
-      name: grafana
-      state: stopped
-  - name: Remove old DBaaS dashboard (Before provisioning times)
-    command: sqlite3 /srv/grafana/grafana.db -cmd ".timeout 60000" "DELETE FROM dashboard WHERE title = 'DBaaS' AND slug = 'dbaas';"
-    changed_when: True
-
-  - name: Remove old PMM Inventory (Before provisioning times)
-    command: sqlite3 /srv/grafana/grafana.db -cmd ".timeout 60000" "DELETE FROM dashboard WHERE title = 'PMM Inventory' AND slug = 'pmm-inventory';"
-    changed_when: True
-
-  - name: Remove old PMM Add Instance dashboard (Before provisioning times)
-    command: sqlite3 /srv/grafana/grafana.db -cmd ".timeout 60000" "DELETE FROM dashboard WHERE title = 'PMM Add Instance' AND slug = 'pmm-add-instance';"
-    changed_when: True
-
-  - name: Remove old PMM Database Checks dashboard (Before provisioning times)
-    command: sqlite3 /srv/grafana/grafana.db -cmd ".timeout 60000" "DELETE FROM dashboard WHERE title = 'PMM Database Checks' AND slug = 'pmm-database-checks';"
-    changed_when: True
+- name: Stop grafana before delete data
+  supervisorctl:
+    name: grafana
+    state: stopped
 
-  - name: Remove old PMM Settings dashboard (Before provisioning times)
-    command: sqlite3 /srv/grafana/grafana.db -cmd ".timeout 60000" "DELETE FROM dashboard WHERE title = 'PMM Settings' AND slug = 'pmm-settings';"
-    changed_when: True
+- name: Remove old dashbaord (Before postgres times)
```

```suggestion
- name: Remove old dashboards (Before postgres times)
```
Why do we remove dashboards before migrating to SQLite?